xpti/CMakeLists.txt

```diff
@@ -1,17 +1,28 @@
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.8)
 
 set(XPTI_VERSION 0.4.1)
+project (xpti VERSION "${XPTI_VERSION}" LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+
 set(XPTI_DIR ${CMAKE_CURRENT_LIST_DIR})
 # Setting the same version as SYCL
 set(CMAKE_CXX_STANDARD 11)
 
+option(XPTI_ENABLE_WERROR OFF)
+
+if (XPTI_ENABLE_WERROR)
+  if(MSVC)
+    set(CMAKE_CXX_FLAGS "/WX ${CMAKE_CXX_FLAGS}")
```

Comment:
If I'm reading this correctly, /WX:NO, if used in the $CMAKE_CXX_FLAGS would override this, while -Wno-error on Linux wouldn't override the flag (because of the order used in each case). This should be consistent.